TypeScript declaration incorrect

When using this lib I get the following TS error: "error TS2309: An export assignment cannot be used in a module with other exported elements."

Currently, I'm having such an error

```
ERROR in ../node_modules/fast-deep-equal/es6/index.d.ts:1:1 - error TS1046: Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.
```

when importing this way

```
import * as equal from 'fast-deep-equal/es6';
```

--- a/.gitignore
+++ b/.gitignore
@@ -66,3 +66,5 @@ yarn.lock
 # generated files
 index.js
 react.js
+
+.idea

--- a/es6/index.d.ts
+++ b/es6/index.d.ts
@@ -1,2 +1,2 @@
-const equal: (a: any, b: any) => boolean;
+declare const equal: (a: any, b: any) => boolean;
 export = equal;

--- a/es6/react.d.ts
+++ b/es6/react.d.ts
@@ -1,2 +1,2 @@
-const equal: (a: any, b: any) => boolean;
+declare const equal: (a: any, b: any) => boolean;
 export = equal;

--- a/react.d.ts
+++ b/react.d.ts
@@ -1,2 +1,2 @@
-const equal: (a: any, b: any) => boolean;
+declare const equal: (a: any, b: any) => boolean;
 export = equal;

